docs/overview/create_available_models.py

```diff
@@ -0,0 +1,155 @@
+"""Updates the available models markdown files."""
+
+from __future__ import annotations
+
+from pathlib import Path
+
+import mteb
+
+START_INSERT = "<!-- START TASK DESCRIPTION -->"
+END_INSERT = "<!-- END TASK DESCRIPTION -->"
+
+model_entry = """
+####  `{model_name}`
+
+**Revision:** `{revision}` • **License:** {license} • [Learn more →]({reference})
```

Comment:
I don't think that revision would be informative
```suggestion
**License:** {license} • [Learn more →]({reference})
```